src/components/VerificationChain/VerificationStep.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Arrow } from "./Arrow";
+
+export interface VerificationStepProps {
+  title: string;
+  description?: string;
+  showArrow?: boolean;
+  isVerifying?: boolean;
+  isVerified?: boolean;
+}
+
+export const VerificationStep: React.FC<VerificationStepProps> = ({
+  title,
+  description,
+  showArrow = false,
+  isVerifying = false,
+  isVerified = false,
```

Comment:
I'd suggest using the same `VerificationStatus` enum defined elsewhere, so you'll probably have to declare it in its own file `VerificationStatus.ts`.

```suggestion
  status: VerificationStatus;
}

export const VerificationStep: React.FC<VerificationStepProps> = ({
  title,
  description,
  showArrow = false,
  status,
```